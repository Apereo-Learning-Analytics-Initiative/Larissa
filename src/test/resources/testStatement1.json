{
  "id" : "12345678-1234-5678-1234-567812345678",
  "actor" : {
    "mbox" : "mailto:xapi@adlnet.gov"
  },
  "verb" : {
    "id" : "http://adlnet.gov/expapi/verbs/created",
    "display" : {
      "en-US" : "created"
    }
  },
  "object" : {
    "id" : "http://example.adlnet.gov/xapi/example/activity"
  }
}