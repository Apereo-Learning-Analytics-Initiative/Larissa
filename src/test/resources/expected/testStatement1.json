{
  "id" : "12345678-1234-5678-1234-567812345678",
  "actor" : {
    "objectType" : "Agent",
    "mbox" : "mailto:xapi@adlnet.gov"
  },
  "verb" : {
    "id" : "http://adlnet.gov/expapi/verbs/created",
    "display" : {
      "en-US" : "created"
    }
  },
  "object" : {
    "objectType" : "Activity",
    "id" : "http://example.adlnet.gov/xapi/example/activity"
  }
}